__tests__/app.test.js

```diff
@@ -173,4 +173,73 @@ describe('app', () => {
             });
         });
     })
-})
\ No newline at end of file
+    describe('POST/api/reviews/:review_id/comments', () => {
+        test('201: responds with added review ', () => {
+            return request(app)
+            .post('/api/reviews/2/comments')
+            .send(
+            {
+                username: 'mallionaire',
+                body: 'great game!'
+            }
+            )
+            .expect(201)
+            .then(({body}) => {
+                const review = body[0]
+            expect(review.author).toBe('mallionaire')
+            expect(review.body).toBe('great game!')
+        })
+    });     
+    });
```

Comment:
Just one test missing here... An extra 201: ignores unnecessary properties. It's possible for our client to post an object with unnecessary key on the post body and these should be ignored